tests/integration/pilot/tunneling_test.go

```diff
@@ -267,12 +275,6 @@ func selectPortName(httpVersion string) string {
 	return "http2-connect"
 }
 
-func getPodIP(ctx framework.TestContext, ns, appSelector string) string {
```

Comment:
Hi @kfaseela, why does `getPodIP` need to remove? I think it's necessary to here for just `pod.Status.PodIP`, any idea?

Reply:
This is not used anywhere else. And I replaced with getPodIPs which can work in both dual-stack and non dual-stack cases. Isn't that enough?

Reply:
pod.Status.PodIPs will work in both dual-stack and non-dual-stack cases, just that it will have a single IP in the second case. The test has taken care of both the scenarios, then why do we have to retain the old func when it is not used anywhere?

Reply:
I'm just thinking about the ambient mode, it may be convenient to just use `pod.Status.PodIP` instead of using  `pod.Status.PodIPs` if Dual Stack still can not support for it.

Reply:
As you may know, not all IP addresses of a pod can support tunnel in Ambient mode.

Reply:
The test currently checks if atleast one of the IPs is present in the tunnel logs, not "all". So not sure what the concern is. And if ambient is not already running this test, we can get back to this and fix, whenever it runs? As of now all the tests pass

Reply:
Okay, I see the `verifyThatRequestWasTunneled` to check all IP addresses one by one until the tunnel support one is found.